src/app/files/file-explorer.js

```diff
@@ -240,15 +244,6 @@ function fileExplorer (localRegistry, files, menuItems) {
 
             if (!removeFolder) {
               tooltip(`failed to remove ${key}. Make sure the directory is empty before removing it.`)
-            } else {
-              const provider = fileManager.currentFileProvider()
-
-              if (provider) {
-                const { type } = provider
-
-                return self.updatePath(type)
```

Comment:
is it not needed anymore? I think this was changed to : `self.updatePath(self.files.type)`

Reply:
Yes, it is no longer needed. I moved it to https://github.com/ethereum/remix-ide/pull/2891/files#diff-4b3872e86385f2fb931b0375fe866ef5R151